src/ILVerification/src/TypeVerifier.cs

```diff
@@ -83,16 +84,19 @@ public void VerifyInterfaces()
                 {
                     VerificationError(VerifierError.InterfaceImplHasDuplicate, type, interfaceType, _module.MetadataReader.GetToken(interfaceHandle));
                 }
+            }
 
+            foreach (InterfaceMetadataObjects implementedInterface in implementedInterfaces)
+            {
                 if (!type.IsAbstract)
-                { 
+                {
                     // Look for missing method implementation
-                    foreach (MethodDesc method in imo.DefType.GetAllMethods())
+                    foreach (MethodDesc method in implementedInterface.DefType.GetAllMethods())
                     {
                         MethodDesc resolvedMethod = virtualMethodAlg.ResolveInterfaceMethodToVirtualMethodOnType(method, type);
                         if (resolvedMethod is null)
                         {
-                            VerificationError(VerifierError.InterfaceMethodNotImplemented, type, imo.DefType, method, _module.MetadataReader.GetToken(_typeDefinitionHandle), _module.MetadataReader.GetToken(imo.InterfaceImplementationHandle), _module.MetadataReader.GetToken(((EcmaMethod)method).Handle));
+                            VerificationError(VerifierError.InterfaceMethodNotImplemented, type, implementedInterface.DefType, method, _module.MetadataReader.GetToken(_typeDefinitionHandle), _module.MetadataReader.GetToken(implementedInterface.InterfaceImplementationHandle), _module.MetadataReader.GetToken(((EcmaMethod)method).Handle));
```

Comment:
> I'm thinking that maybe we could remove completely "tokens"

I agree that we should not be dealing with tokens explicitly for each error like it is done right now. And the default error formatting should be without tokens since they are not very useful for regular use.

The tokens are sometimes useful e.g. for investigations of obfuscated code. I think the best way to deal with that would be to have a centralized methods to format method into string. By default, this method would produce regular name without token. And when somebody species `--tokens` command line option for PEVerify, the method would produce name and token.

I do think you need to do this while thing in this PR. You can just do the prep-work to introduce the centralized methods to format the type system item into string to fix the immediate problem that you are hitting.

Reply:
> I do think you need to do this whole thing in this PR. You ...

You meant `I don't think`?

Reply:
Right ... I make this typo a the time.

Reply:
I removed tokens from defaul output and added simple format helper. The idea is to concat token value when `--tokens` is specified, the message will be something like:
```
[MD]: Error: Interface implementation has a duplicate. Class '[InterfaceImplementation]InterfaceImplHasDuplicate_InvalidType_InterfaceImplHasDuplicate(0x12345)' Interface: '[InterfaceDefinition]Interface(0x6789)'
```
and so on...what do you think?